### PR TITLE
Cards and about section

### PR DESCRIPTION
**Pull request by J-C-S-V**
In this project I did the following:

- Create linters
- Use .gitignore to untrack files
- Add HTML and CSS files
- Use feature branches
- Use cards for the projects
- Create an about section
- Create a mobile first page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,5 +59,163 @@ <h3>LET'S CONNECT</h3>
...
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="skills">
+            <div>html</div>
+            <div>css</div>
+            <div>javascript</div>
+          </div>
```
Comment (nahnahdev) [inline index.html]: - [x] Great job working on your project. :+1:  I noticed here that you didn't make use of the unordered list tags as suggested in the Project Requirement. Kindly change the (`div` tags) you used here to (`ul` and `li` tags) so as to meet all the requirements. :+1:
- [x] Currently, the `font-size` is different from what was in the design guidelines so kindly change this as well :+1: 
- [x] Please repeat this for all the other cards :heavy_check_mark: 
- [x] You can read more about  (`ul` and `li` tags) [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) :100: 
- Example
```
<ul>
  <li>Coffee</li>
  <li>Tea</li>
  <li>Milk</li>
</ul>

```
- Project Requirement 
image

File: index.html [hunk 2]
```diff
@@ -59,5 +59,163 @@ <h3>LET'S CONNECT</h3>
...
       </section>
     </main>
+    <!-- Cards section -->
+    <section class="main-container-projects">
+      <div class="cards card-1">
+        <div class="main-img">
+          <img src="./img/Snapshoot Portfolio.png" alt="person" />
+        </div>
+        <article class="second-container">
+          <h2>Tonic</h2>
```
Comment (nahnahdev) [inline index.html:69]: - [x] I will suggest that you apply all the styles that were assigned to this header tag in the Figma design. I noticed that you only applied the font color and font-family. So kindly apply the rest as well. Please try to stick to the Figma design as much as possible :wink: 
- Figma styles
image
- Your styles
image

File: index.html [hunk 3]
```diff
@@ -59,5 +59,163 @@ <h3>LET'S CONNECT</h3>
...
+          <div class="horizontal-section">
+            <span>CANOPY</span>
+            <input type="radio" checked />Back End Dev
+            <input type="radio" checked />2015
+          </div>
```
Comment (nahnahdev) [inline index.html]: - [x] I also noticed that your `horizontal-section` doesn't look completely like the Figma Design. Kindly try to stick to the Figma design as much as possible so please fix this as well :+1: 
- Your Design
image
-The Figma Design
image

File: index.html [hunk 4]
```diff
@@ -59,5 +59,163 @@ <h3>LET'S CONNECT</h3>
...
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
```
Comment (nahnahdev) [inline index.html:79]: - [x] Kindly apply the right `font-size` to this paragraph as well :100: Please refer to the Design guidelines :+1:

File: index.html [hunk 5]
```diff
@@ -59,5 +59,163 @@ <h3>LET'S CONNECT</h3>
...
+          <div class="skills">
+            <div>html</div>
+            <div>css</div>
+            <div>javascript</div>
+          </div>
+          <button class="see-project">See Project</button>
```
Comment (nahnahdev) [inline index.html:85]: - [x] Please apply the right button styles to all of your buttons. This is essential for user interactivity and to meet all of the design guidelines. 
- [x] You only added `hover` styles but it doesn't match the Figma design so, please  fix this as well and include the other button styles :100:  
- Button Interaction Styles 
image
- Your button when Pressed and on Hover
[Screencast from 02-17-2023 04:39:08 AM.webm](https://user-images.githubusercontent.com/102699958/219551334-a0778061-bebf-4db3-9815-d4b51fa69506.webm)

File: index.html [hunk 6]
```diff
@@ -59,5 +59,163 @@ <h3>LET'S CONNECT</h3>
...
+        </article>
+      </div>
+      <!-- About section -->
+      <section class="about">
+        <h2>
+          About <br />
+          Myself
+        </h2>
```
Comment (nahnahdev) [inline index.html:169]: - [x] Kindly increase the margin top above the (`h2` tag) so you design looks more like the Figma Design :+1: 
- Figma Design
image
- Your Design
image

File: index.html [hunk 7]
```diff
@@ -59,5 +59,163 @@ <h3>LET'S CONNECT</h3>
...
+          </div>
+          <div class="skills-boxed">
+            <img src="/img/HTML.png" alt="HTML" />HTML
+          </div>
+          <div class="skills-boxed">
+            <img src="/img/CSS3.png" alt="CSS3" />CSS
+          </div>
+          <div class="frameworks-skills">Frameworks <span>></span></div>
+          <hr />
+          <div class="frameworks-skills">Skills<span>></span></div>
```
Comment (nahnahdev) [inline index.html]: - [x] In this part of your project, according to the project requirements you are to use the (`ul` ad `li` tags).
image
- [x] So please fix this as well and ensure that the `Languages ` container is vertically aligned  with the `Frameworks` and `Skills` and have the right styles applied as per the Figma Design :+1: 
- [x] The icons are also not well styled at the moment so please style them according to the Figma design :100: 
- Your Design 
image
 - The Figma Design
 
image

File: index.html [hunk 8]
```diff
@@ -59,5 +59,163 @@ <h3>LET'S CONNECT</h3>
...
+            sign-ups required.
+          </p>
+          <div class="skills">
+            <div>html</div>
+            <div>css</div>
+            <div>javascript</div>
+          </div>
+          <button class="see-project">See Project</button>
+        </article>
+      </div>
```
Comment (nahnahdev) [inline index.html]: - [x] Please note that the minimum mobile screen size should be `375`px so your mobile design should be responsive from that size until the desktop screen size but from the video below, you can notice that from `390`px it wasn't responsive all through to `482` px before it began to look okay. So kindly ensure that from your design is responsive from '375' px and above. :+1: 
- Video Reference
[Screencast from 02-17-2023 05:04:26 AM.webm](https://user-images.githubusercontent.com/102699958/219554565-a9010c2f-bd9a-47cc-b35d-13ece97553a3.webm)

**Review by nahnahdev — CHANGES_REQUESTED** [review]
Hi @J-C-S-V , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- [x] Professional README :heavy_check_mark: 
- [x] Good use of GitHub flow :heavy_check_mark: 
- [x] No linter errors :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 7 above)
Comment (J-C-S-V) [reply index.html]: Hello @NahnahAJ,

Could you please go deeply on this suggestion? 'The icons are also not well styled at the moment so please style them according to the Figma design', It is too general and I don't understand what is wrong with the svg icons.

Thanks for your help!

File: index.html [hunk 9]
```diff
@@ -59,5 +59,170 @@ <h3>LET'S CONNECT</h3>
...
     </main>
+    <!-- Cards section -->
+    <section class="main-container-projects">
+      <div class="cards card-1">
+        <div class="main-img">
+          <img src="./img/Snapshoot Portfolio.png" alt="person" />
+        </div>
+        <article class="second-container">
+          <h2>Tonic</h2>
+          <div class="horizontal-section">
```
Comment (MeqdamAlqudah) [inline index.html:70]: - [Optional ] please fix this weried space in the right , I suggest you to remove the padding around the about section it is not helping.
image
image

- please fix the space in the left of the about section it should match the design
image

File: index.html [hunk 10]
```diff
@@ -59,5 +59,170 @@ <h3>LET'S CONNECT</h3>
...
+            <span>CANOPY</span>
```
Comment (MeqdamAlqudah) [inline index.html:71]: - [Optional] Please fix the position of the background image it is up a little bit 
image

File: index.html [hunk 11]
```diff
@@ -59,5 +59,170 @@ <h3>LET'S CONNECT</h3>
...
+            <img class="bullet-img" src="./img/bullet.svg" alt="bullet" />Back
```
Comment (MeqdamAlqudah) [inline index.html:72]: - [Optional] Please make sure to add this corner to your headline section
image
image

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @J-C-S-V , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlight 
:+1: no linter errors 
:+1: you have done all the required changes from the last reviewer 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**